README.md

```diff
@@ -82,7 +93,7 @@ func (p *PostResource) Get(r *http.Request) (int, gorest.Response) {
     }
 
     // Insert the post in a Response object
-    response := NewResponse()
+    response := gorest.NewStandardResponse()
```

Comment:
The usage of the non-existing `NewResponse` function in this example was thought in order to force the reader to think about the possibility of creating their own responses just by implementing the `Response` interface. 

If this wasn't clear I'll try to better explain myself on this matter, as of now it is ok to use `NewStandardResponse` inside the example, I'll do my change elsewhere ;)

Reply:
Yeah if it was intended I suggest to explain better this thing.